src/drivers/optical_flow/paa3905/PAA3905.cpp

```diff
@@ -393,6 +393,8 @@ void PAA3905::RunImpl()
 				const bool motion_reported = (buffer.data.Motion & Motion_Bit::MotionOccurred);
 
 				if (data_valid) {
+					sensor_optical_flow.quality = 0;
```

Comment:
isn't it already 0 here?

Reply:
If you just wanted to make it explicit for readability you could do it below if !motion_reported.

Reply:
Updated and force pushed.